### PR TITLE
Chore | Add Codowners file

### PR DESCRIPTION
**Pull request by EstebanBorai**
As we are just 3 - 4 collaborators working on this project at the moment
awaiting on 2 reviews for merging is turning a bit of a blocker.

In order to keep moving on perhaps defining code owners while we grow
is a good idea, so all PRs are reviewed by more active members instead of
waiting for 2 code reviewers. (we usually have just one).

To keep the structure of the project and provide the most of the feedback
before officially merging a PR I think that its important to keep PR reviews from
at least 1 collaborator.

Suggestions on **this or other** approach would be awesome I think is important
to avoid the [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life) kind model and move forward to a _open governance_.

If you want to be a code owner just edit the CODEOWNERS file but keep in mind that
your review will be required in future PRs!

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Understand this file here: https://github.blog/2017-07-06-introducing-code-owners/#how-code-owners-work
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+*    @EstebanBorai
```
Comment (morenol) [inline .github/CODEOWNERS:6]:
```suggestion
*    @EstebanBorai
```
```suggestion
*    @EstebanBorai
*    @morenol 
```

**Comment by EstebanBorai** [comment]
@morenol should I merge this now?

**Review by EstebanBorai — COMMENTED** [review]
Awesome! Lets wait and see if anyone else want to be in the list!

**Review by morenol — APPROVED** [review]
It looks like nobody wants to be on the list.  😂 

LGTM! 

I think that you can merge this now.

**Pull request merged**